examples/event-sourced-child-entity/Parts.php

```diff
@@ -141,7 +141,7 @@ class PartManufacturerWasRenamedEvent
 
     public function __construct($partId, $manufacturerName)
     {
-        $this->partId = $partId;
+        $this->partId           = $partId;
```

Comment:
That's part of the php_cs definition that Broadway uses. 

Some of us prefer having them lined up, some don't. But the broadway team chose to put that in their coding standards file. Personal preference is irrelevant.